.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "bao-ci"]
+	path = bao-ci
```

Comment:
The convention stipulated was that the path of the repo would only be `ci`

Reply:
Fixed [a0ff278](https://github.com/bao-project/bao-docs/pull/68/commits/a0ff27868a276daf8cdd400a82096f2d7e121683)